zipline/protocol.py

```diff
@@ -360,17 +367,27 @@ def minute_get_bars(days, env=None):
         return self._get_bars(days)
 
     def mavg(self, days):
-        return self._get_buffer(self._get_bars(days)).mean()
+        bars = self._get_bars(days)
```

Comment:
Just a style (and minor perf) comment about these functions, I find caching all these intermediate results in the locals harder to read than just chaining the computations. Chaining the computations also keeps all the data on the python stack instead of bouncing between the fast and the stack constantly.